Project.toml

```diff
@@ -6,9 +6,11 @@ desc = "Chain types and utility functions for MCMC simulations."
 version = "4.0.0-DEV"
 
 [deps]
+AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 AxisArrays = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
```

Comment:
I think we don't need to (and actually don't want to) include `FFTW` as dependency. Everything can be implemented using the interface in `AbstractFFTs`, and then users can load `FFTW` (or an alternative implementing the interface) if they want to use the FFT method.

Reply:
```suggestion
```

Reply:
BTW it's fine (and necessary) to include `FFTW` as test dependency.